Cannot insert to a table with only an autoincrement id

``` go
type Account struct {
	Id int64 `xorm:"pk autoincr"`
}

engine.Insert(&Account{})
```

Error

```
panic: runtime error: slice bounds out of range

goroutine 27 [running]:
github.com/go-xorm/xorm.(*Session).innerInsert(0xc42033b8c0, 0x13f2fe0, 0xc420360520, 0x0, 0x0, 0x0)
	/Users/suyejun/go/src/github.com/go-xorm/xorm/session_insert.go:346 +0x24e8
github.com/go-xorm/xorm.(*Session).Insert(0xc42033b8c0, 0xc4201c1810, 0x1, 0x1, 0x0, 0x0, 0x0)
	/Users/suyejun/go/src/github.com/go-xorm/xorm/session_insert.go:49 +0x325
github.com/go-xorm/xorm.(*Engine).Insert(0xc420114140, 0xc4201c1810, 0x1, 0x1, 0x0, 0x0, 0x0)
	/Users/suyejun/go/src/github.com/go-xorm/xorm/engine.go:1394 +0xa4
....
...
exit status 2
```

--- a/session_insert.go
+++ b/session_insert.go
@@ -343,15 +343,26 @@ func (session *Session) innerInsert(bean interface{}) (int64, error) {
 	if len(exprColVals) > 0 {
 		colPlaces = colPlaces + strings.Join(exprColVals, ", ")
 	} else {
-		colPlaces = colPlaces[0 : len(colPlaces)-2]
+		if len(colPlaces) > 0 {
+			colPlaces = colPlaces[0 : len(colPlaces)-2]
+		}
 	}
 
-	sqlStr := fmt.Sprintf("INSERT INTO %s (%v%v%v) VALUES (%v)",
-		session.Engine.Quote(session.Statement.TableName()),
-		session.Engine.QuoteStr(),
-		strings.Join(colNames, session.Engine.Quote(", ")),
-		session.Engine.QuoteStr(),
-		colPlaces)
+	var sqlStr string
+	if len(colPlaces) > 0 {
+		sqlStr = fmt.Sprintf("INSERT INTO %s (%v%v%v) VALUES (%v)",
+			session.Engine.Quote(session.Statement.TableName()),
+			session.Engine.QuoteStr(),
+			strings.Join(colNames, session.Engine.Quote(", ")),
+			session.Engine.QuoteStr(),
+			colPlaces)
+	} else {
+		if session.Engine.dialect.DBType() == core.SQLITE || session.Engine.dialect.DBType() == core.POSTGRES {
+			sqlStr = fmt.Sprintf("INSERT INTO %s DEFAULT VALUES", session.Engine.Quote(session.Statement.TableName()))
+		} else {
+			sqlStr = fmt.Sprintf("INSERT INTO %s VALUES ()", session.Engine.Quote(session.Statement.TableName()))
+		}
+	}
 
 	handleAfterInsertProcessorFunc := func(bean interface{}) {
 		if session.IsAutoCommit {
@@ -384,7 +395,6 @@ func (session *Session) innerInsert(bean interface{}) (int64, error) {
 	// for postgres, many of them didn't implement lastInsertId, so we should
 	// implemented it ourself.
 	if session.Engine.dialect.DBType() == core.ORACLE && len(table.AutoIncrement) > 0 {
-		//assert table.AutoIncrement != ""
 		res, err := session.query("select seq_atable.currval from dual", args...)
 		if err != nil {
 			return 0, err

--- a/session_pk_test.go
+++ b/session_pk_test.go
@@ -1105,3 +1105,15 @@ func TestMyStringId(t *testing.T) {
 		panic(err)
 	}
 }
+
+func TestSingleAutoIncrColumn(t *testing.T) {
+	type Account struct {
+		Id int64 `xorm:"pk autoincr"`
+	}
+
+	assert.NoError(t, prepareEngine())
+	assertSync(t, new(Account))
+
+	_, err := testEngine.Insert(&Account{})
+	assert.NoError(t, err)
+}

